en/lessons/advanced/error-handling.md

```diff
@@ -11,6 +11,17 @@ This lesson will focus on interacting with the latter.
 
 {% include toc.html %}
 
+## General conventions
+
+At the moment, Elixir community has come to some conventions regarding returning errors:
+
+* For errors that are part of the regular operation of a function (e.g. a user entered a wrong type of date), a function returns {:ok, result} and {:error, reason} accordingly.
+* For errors that are not part of normal operations (e.g. being unable to parse configuration data) you throw an exception.
+
+We generally handle standard flow errors by [Pattern Matching](../basics/pattern-matching/), but in this lesson, we are focusing on the second case - on the exceptions.
+
+Usually, in public APIs, you can also find a second version of the function with a ! (example!/1) that returns the unwrapped result or raises an error.
```

Comment:
This is kind of minor but maybe we can say `Often` instead of `Usually`. I'm not sure that I actually do see this pattern super often, although I am a fan of implementing it. Basically just not sure how much of a community-wide standard it is so want to use slightly less strong language. Let me know if you disagree!